verification/verify_play_page.py

```
@@ -0,0 +1,24 @@
+    with sync_playwright() as p:
+        browser = p.chromium.launch()
+        page = browser.new_page()
+
+        print("Navigating to Play page...")
+        page.goto("http://localhost:5173/play")
+
+        print("Waiting for scene text OR error banner...")
+        try:
+            # Wait for either scene text or error banner
+            page.wait_for_selector(".scene-text, .error-banner", timeout=10000)
+        except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)